pkg/ssh/ssh.go

```diff
@@ -121,10 +122,11 @@ func (s *SSHTunnel) Open() error {
 	return err
 }
 
-func (s *SSHTunnel) Dial(network, address string) (net.Conn, error) {
+func (s *SSHTunnel) Dial(ctx context.Context, network, address string) (net.Conn, error) {
 	if s.client == nil {
 		return nil, errors.New("tunnel is not opened.")
 	}
+	// Cannot use context here
```

Comment:
is this a todo?

Reply:
No, just a comment, pointing out the sad fact that the library used here (`golang.org/x/crypto/ssh`) does not allow to pass a context for dial.

Reply:
maybe be a bit more explicit in the comment.